src/cipher/encryptingstorage/encrypt_util.py

```diff
@@ -72,9 +85,15 @@ def encryptBytes(self, data):
     def decryptBytes(self, data):
         try:
             return self.facility.decrypt(self.key, data)
-        except M2Crypto.EVP.EVPError:
+        except:
```

Comment:
Haven't found a valid replacement for `M2Crypto.EVP.EVPError`, will look again.

Thanks for your response.